process/block/metablock.go

```diff
@@ -86,7 +86,13 @@ func NewMetaProcessor(arguments ArgMetaProcessor) (*metaProcessor, error) {
 		return nil, process.ErrNilEpochStartSystemSCProcessor
 	}
 
-	pruningDelay := uint32(arguments.Config.StateTriesConfig.PeerStatePruningQueueSize * pruningDelayMultiplier)
+	pruningQueueSize := arguments.Config.StateTriesConfig.PeerStatePruningQueueSize
+	pruningDelay := uint32(pruningQueueSize * pruningDelayMultiplier)
+	if pruningDelay < defaultPruningDelay {
+		log.Warn("using default pruning delay", "peer state pruning queue size", pruningQueueSize)
```

Comment:
I think this will be used also on the metachain's user account. Suggestion to change the message accordingly.

Reply:
Done